services/newsfeed/config_twitter.js

```diff
@@ -14,45 +14,35 @@
  *
  */
 const getTwitterConfig = () => ({
+	name: 'twitter_lisk',
+	table: 'newsfeed',
 	type: 'twitter',
 	interval: 5 * 60, // seconds
 	url: 'statuses/user_timeline',
 	requestOptions: {
 		screen_name: 'liskHQ',
-		count: 20,
+		count: 30,
+		exclude_replies: true,
+		include_rts: true,
 	},
 	newsfeed: {
 		mapper: {
-			source: '=,string',
-			source_id: 'id,string',
 			hash: '=,string',
-			content: 'text,string',
-			url: '=,string',
-			timestamp: '=',
 			author: '=,string',
+			content_orig: 'text,string',
+			content_t: 'text,string',
```

Comment:
There is no custom mapper for `content_t` (truncated text). Should `content_t` and ``content_orig` be the same in case of Twitter?

Reply:
`content_orig` and `content_t` are mapped to the same field `text` within Twitter API the response.